examples/vos0.rs

```diff
@@ -0,0 +1,49 @@
+#![deny(warnings)]
+#![deny(unsafe_code)]
+#![no_main]
+#![no_std]
+
+extern crate panic_itm;
+
+use cortex_m;
+use cortex_m_rt::entry;
+use stm32h7xx_hal::{pac, rcc, prelude::*};
+
+use cortex_m_log::println;
+use cortex_m_log::{
+    destination::Itm, printer::itm::InterruptSync as InterruptSyncItm,
+};
+
+#[entry]
+fn main() -> ! {
+    let cp = cortex_m::Peripherals::take().unwrap();
+    let dp = pac::Peripherals::take().expect("Cannot take peripherals");
+    let mut log = InterruptSyncItm::new(Itm::new(cp.ITM));
+
+    // Constrain and Freeze power
+    println!(log, "Setup PWR...                  ");
+    let pwr = dp.PWR.constrain();
+    let rcc = dp.RCC.constrain();
+    let vos = pwr.freeze_vos0(&rcc, &dp.SYSCFG);
+
+    // Constrain and Freeze clock
+    println!(log, "Setup RCC...                  ");
+    let ccdr = rcc
+        .sys_ck(480.mhz())
+        .pll1_strategy(rcc::PllConfigStrategy::Iterative)
+        .freeze(vos, &dp.SYSCFG);
+
+    println!(log, "");
+    println!(log, "stm32h7xx-hal example - RCC");
+    println!(log, "");
+
+    // HCLK
+    println!(log, "hclk = {} MHz", ccdr.clocks.hclk().0 as f32 / 1e6);
+    // assert_eq!(ccdr.clocks.hclk().0, 50_000_000);
```

Comment:
Worth setting the new value and uncommenting this line

Reply:
Yes, I just forgot.